optaplanner-7-benchmarks/optaplanner-perf-framework/src/main/java/org/jboss/qa/brms/performance/examples/conferencescheduling/ConferenceSchedulingExample.java

```diff
@@ -33,7 +33,7 @@
 public final class ConferenceSchedulingExample extends AbstractExample<ConferenceSolution> {
 
     public static final String DRL_FILE =
-            "org/optaplanner/examples/conferencescheduling/optional/score/conferenceSchedulingConstraints.drl";
+            "org/optaplanner/examples/conferencescheduling/solver/conferenceSchedulingConstraints.drl";
```

Comment:
Please check also the XML configuration for this example:
https://github.com/kiegroup/kie-benchmarks/blob/main/optaplanner-7-benchmarks/optaplanner-perf-framework/src/main/resources/org/jboss/qa/brms/performance/examples/conferencescheduling/solver/conferenceSchedulingSolverConfig.xml#L8

Reply:
A bit confused, isn't it the next file that you can see bellow?

Reply:
You're right, I have mistaken it with a different file. Please disregard this comment.